mne/preprocessing/tests/test_scalp_coupling_index.py

```diff
@@ -49,12 +49,6 @@ def test_scalp_coupling_index(fname, fmt, tmpdir):
     # Set next two channels to anti correlation
     raw._data[4] = new_data
     raw._data[5] = new_data * -1.0
-    # Set next two channels to be uncorrelated
-    # TODO: this might be a bad idea as sometimes random noise might correlate
-    raw._data[6] = new_data
-    raw._data[7] = np.random.rand(raw._data[0].shape[0])
```

Comment:
that's why you should not use np.random but you should rather use a random state. So:

`raw._data[7] = np.random.RandomState(42).rand(raw._data[0].shape[0])`